bootstrap.sh

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash
 kubectl apply -f .infrastructure/mysql/ns.yml
+kubectl apply -f .infrastructure/app/ns.yml
+
+kubectl apply -f .infrastructure/security/rbac.yml
```

Comment:
The RBAC configuration is critical for security. Double-check that the roles and permissions defined in `rbac.yml` are appropriate for your application's needs.